server.py

```
@@ -37,7 +37,17 @@
 # Global Variables
 # TODO: SLACK_INTEGRATION_POINT - Variáveis globais para integração com Slack
 agent = None  # Agente OpenAI principal
-agent_lock = asyncio.Lock()  # Lock para concorrência (removido para permitir múltiplos usuários)
+
+# Concurrency semaphore for multi-user handling
+import math
+try:
+    max_concurrency = int(os.environ.get("LIVIA_MAX_CONCURRENCY", "5"))
+    if max_concurrency &lt; 1:
```

Ruff (0.11.9)

SyntaxError: Expected ':', found ';'

---

Multiple statements on one line (semicolon)

(E702)